pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -277,6 +284,19 @@ func (dswp *desiredStateOfWorldPopulator) findAndRemoveDeletedPods() {
 		dswp.desiredStateOfWorld.DeletePodFromVolume(
 			volumeToMount.PodName, volumeToMount.VolumeName)
 		dswp.deleteProcessedPod(volumeToMount.PodName)
+		// Process volume spec for each volume defined in pod
+		if pod != nil && len(pod.Spec.Volumes) > 0 {
+			for _, podVolume := range pod.Spec.Volumes {
+				if pvcSource :=
```

Comment:
Nit: I personally find the `if` statement being split up over multiple lines a little difficult to read. If we assign `pvcSource` in the previous line, can we keep the `if` statement in one line?